### PR TITLE
Add new first-spore tutorial

### PR DESCRIPTION
**Pull request by ShookLyngs**
### Changes
- Add new first-spore tutorial: "Creating your first spore"

### CODE REVIEW

File: docs/tutorials/create-first-spore/create-spore.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 2
+---
+
+# Create your first spore on-chain
```
Comment (anipenguin) [inline docs/tutorials/create-first-spore/create-spore.mdx:5]:
```suggestion
# Create Spore
```

File: docs/tutorials/create-first-spore/index.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 1
+---
+
+import EmbedCard from "@site/src/components/EmbedCard";
+
+# Creating your first spore
```
Comment (anipenguin) [inline docs/tutorials/create-first-spore/index.mdx:7]:
```suggestion
# Creating Your First Spore
```

File: docs/tutorials/create-first-spore/index.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  className="margin-top--sm margin-bottom--lg"
+/>
+
+This spore, hosted on the Nervos CKB Testnet, stores an image directly on the blockchain. With Spore Protocol, you have *full* control over the *entire* content, not just a URL to where the content is stored often off-chain.
+
+If this piques your interest, let's dive into this step-by-step guide to:
+
+1. [Set up an account on Nervos CKB](./setup-account)
+2. [Create your first spore on-chain](./create-spore)
+3. [Explore the code](./explore-code)
```
Comment (anipenguin) [inline docs/tutorials/create-first-spore/index.mdx:27]:
```suggestion
1. [**Set up an account** on Nervos CKB](./setup-account)
2. [**Create your first spore** on-chain](./create-spore)
3. [**Explore the code**](./explore-code)
```

File: docs/tutorials/create-first-spore/index.mdx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Let's begin by creating an account on Nervos CKB: [Set up an account on Nervos CKB](./setup-account).
```
Comment (anipenguin) [inline docs/tutorials/create-first-spore/index.mdx:29]: Screenshot 2023-10-18 at 12 02 53 AM
having both is redundant?

File: docs/tutorials/create-first-spore/explore-code.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
+---
+sidebar_position: 3
+---
+
+# Explore the code
```
Comment (anipenguin) [inline docs/tutorials/create-first-spore/explore-code.md:5]:
```suggestion
# Explore the Code
```

File: docs/tutorials/create-first-spore/index.mdx (hunk 4 above)
Comment (ShookLyngs) [reply docs/tutorials/create-first-spore/index.mdx:29]: Do you think we should remove the first one?

File: docs/tutorials/create-first-spore/index.mdx (hunk 3 above)
Comment (ShookLyngs) [reply docs/tutorials/create-first-spore/index.mdx:27]: They look messy on the page, do you want to bold the whole list instead?

**Review by anipenguin — APPROVED** [review]

**Pull request merged**